software/java/en.md

```diff
@@ -14,42 +14,42 @@ You can set up Java by following the instructions below:
 
 ## JRE
 
-Grab the Java Runtime Environment (JRE) as `.tar.gz` from the [Oracle Download Page](https://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html).
+Grab the newest Java Runtime Environment (JRE) as `.tar.gz` from the [Oracle Download Page](https://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html).
 
 Extract JRE and move it to /opt:
 
 ``` bash
 cd ~/Downloads
-tar xf jre-8u181-linux-x64.tar.gz
+tar xf jre-8u202-linux-x64.tar.gz
```

Comment:
Should just try using wildcards / globbing instead. `jre-8u*-linux-x64.tar.gz`

Reply:
Done.